pymc/distributions/discrete.py

```diff
@@ -920,6 +920,12 @@ def dist(cls, N, k, n, *args, **kwargs):
         n = at.as_tensor_variable(intX(n))
         return super().dist([good, bad, n], *args, **kwargs)
 
+    def get_moment(rv, size, N, k, n):
+        mode = intX(at.floor((n + 1) * (k + 1) / (N + 2)))
```

Comment:
```suggestion
        mode = at.floor((n + 1) * (k + 1) / (N + 2))
```

Should be fine without intX, we take care of that in the method that calls `get_moment`

Reply:
Sure, I removed it in the latest commit.